### PR TITLE
Us38 sensor rpi com data storage

### PR DESCRIPTION
**Pull request by tcyubahi**

### CODE REVIEW

File: Guides/MongoDBSetUp.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+		```exit```
+
+5. Install the Python MongoDB driver (PyMongo)
+
+	```python3 -m pip install pymongo==3.4.0```
+
+	Opted to use 3.4.0 because the latest (3.7.2) was having problems connecting with the latest mongo release
+
+6. Run the python daemon from folder SensorReader, sensorReader.py, and make sure there is data reported 	at least once so you can test the next step
```
Comment (smwatki2) [inline Guides/MongoDBSetUp.md]: What do you mean here? I try to run `python3 sensorReader.py` but it just stays at "Connection to Blocker established 0", and nothing shows up in the mongoDB when I check the next step...
image

Comment (tcyubahi) [reply Guides/MongoDBSetUp.md]: Did you run the daemon?

Comment (smwatki2) [reply Guides/MongoDBSetUp.md]: I'm able to get a reading when I run `python3 /opt/miflora-mqtt-daemon/miflora-mqtt-daemon.py`, but running `python3 sensorReader.py` and `python3 miflora-mqtt-daemon.py` gives me the error: 
`Traceback (most recent call last):
  File "miflora-mqtt-daemon.py", line 129, in <module>
    reporting_mode = config['General'].get('reporting_method', 'mqtt-json')
  File "/usr/lib/python3.5/configparser.py", line 956, in __getitem__
    raise KeyError(key)
KeyError: 'General'`

**Comment by smwatki2** [comment]
I still can't get the two talking to each other. untitled

**Pull request merged**